### PR TITLE
Allow flowcontrol/v1beta3

### PR DESCRIPTION
**Pull request by stone-z**
### This PR:

- allows the chart to detect and use the flowcontrol v1beta3 API

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

File: helm/trivy-operator/templates/priority-level-configuration.yaml [hunk 1]
```diff
@@ -9,7 +11,7 @@ metadata:
 spec:
   type: Limited
   limited:
-    assuredConcurrencyShares: 5
+    nominalConcurrencyShares: 5
```
Comment (fhielpos) [inline helm/trivy-operator/templates/priority-level-configuration.yaml:14]: Does this work across all apiVersions?

**Review by fhielpos — APPROVED** [review]
LGTM if it works with every version

File: helm/trivy-operator/templates/priority-level-configuration.yaml (hunk 1 above)
Comment (stone-z) [reply helm/trivy-operator/templates/priority-level-configuration.yaml:14]: No, but I'm fine not supporting the older versions on the next app version. The app should be installed via the bundle anyway.
App version v0.9.1 --> bundle version 1.7.2 and 1.8.1 --> releases v28+ so k8s 1.28+ with v1beta3. Should be all good

Comment (fhielpos) [reply helm/trivy-operator/templates/priority-level-configuration.yaml:14]: All good then 😄

**Comment by stone-z** [comment]
Tested the full upgrade path from v25

**Pull request merged**